R/aaa-auto.R

```diff
@@ -3364,6 +3364,29 @@ modularity_matrix_impl <- function(graph, weights=NULL, resolution=1.0, directed
   res
 }
 
+community_leading_eigenvector_impl <- function(graph, weights=NULL, steps=-1, options=arpack_defaults(), start=FALSE, callback=NULL) {
```

Comment:
this function has no `extra` argument.

- I wonder how to pass it.
- I wonder whether it means I shouldn't try auto-generating this function. :sweat_smile: